avanza/models/market.go

```diff
@@ -4,6 +4,26 @@ type GetMarketDataParams struct {
 	OrderBookID string `path:"orderbookID" validate:"required"`
 }
 
+type GetChartDataResponse struct {
+	Ohlc []struct {
+		Timestamp         int64   `json:"timestamp"`
+		Open              float64 `json:"open"`
+		Close             float64 `json:"close"`
+		Low               float64 `json:"low"`
+		High              float64 `json:"high"`
+		TotalVolumeTraded int     `json:"totalVolumeTraded"`
```

Comment:
Depending on the ticker, this field can be a int64. However go don't handle int64 scientific unmarhal (see https://github.com/golang/go/issues/5562). So i suggest just changing this to a float64

Reply:
Have you seen if the avanza api returns this field in scientific format?
with float64 the csv looks like: 
2023-01-10,130.13,131.17,129.92,128.16,5.0560696E+07
with int64 the csv lookslike:
2023-01-10,130.13,131.17,130.01,128.16,50782424

If it works I think int (or int64) is better.

Reply:
I agree that int64 is better. Not sure if Avanza is returning scientific format, some JSON api implementations do it by default since it supported by the JSON format. But i guess we can use int64 until someone is reporting errors! :+1: